build_tools/cmake/build_android.sh

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the IREE project with CMake. Designed for CI, but can be run manually.
+# This uses previously cached build results and does not clear the build
+# directory. For a clean build, use build_tools/cmake/clean_build.sh
```

Comment:
`clean_build.sh` runs `rebuild.sh`, so we probably shouldn't mention it here?

Reply:
Oops. Inherited from the copied file. This is not the case for Android. Deleted.